common/test/acceptance/pages/studio/video/video.py

```diff
@@ -48,6 +48,10 @@
     'collapse_link': '.collapse-action.collapse-setting',
 }
 
+VIDEO_MENUS = {
+    'transcript-skip': '.sr-is-focusable.transcript-start'
```

Comment:
Same here. I'd add a comma at the end of the last (only) line in this dict. (See [why-does-python-allow-commas-at-the-end-of-lists-and-tuples](https://docs.python.org/2/faq/design.html#why-does-python-allow-commas-at-the-end-of-lists-and-tuples).)